index.bs

```diff
@@ -302,8 +303,9 @@ Each {{WebSocket}} object has an associated <dfn>url</dfn> (a [=URL record=]).
 
 <div algorithm>
 
- The <dfn constructor for=WebSocket>WebSocket(|url|, |protocols|)</dfn> constructor, when invoked,
- must run these steps:
+ The <dfn constructor for="WebSocket" data-lt="WebSocket(url, protocols)"><code
+ data-x="dom-WebSocket">new WebSocket(<var>url</var>, <var>protocols</var>)</code></dfn>
```

Comment:
I think `data-x` is an HTML spec attribute that is not used by Bikeshed. Also, is it okay to use `|url|` instead of `<var>url</var>`?

Reply:
Done.